modules/orbitsoftBidAdapter.js

```diff
@@ -0,0 +1,232 @@
+import { getBidRequest } from 'src/utils';
+
+let CONSTANTS = require('src/constants');
+let bidmanager = require('src/bidmanager');
+let bidfactory = require('src/bidfactory');
+let adloader = require('src/adloader');
+let utils = require('src/utils');
+let adaptermanager = require('src/adaptermanager');
+let Adapter = require('src/adapter').default;
+
+let ORBITSOFT_BIDDERCODE = 'orbitsoft';
+let styleParamsToFieldsMap = {
+  'title.family': 'f1', // headerFont
+  'title.size': 'fs1', // headerFontSize
+  'title.weight': 'w1', // headerWeight
+  'title.style': 's1', // headerStyle
+  'title.color': 'c3', // headerColor
+  'description.family': 'f2', // descriptionFont
+  'description.size': 'fs2', // descriptionFontSize
+  'description.weight': 'w2', // descriptionWeight
+  'description.style': 's2', // descriptionStyle
+  'description.color': 'c4', // descriptionColor
+  'url.family': 'f3', // urlFont
+  'url.size': 'fs3', // urlFontSize
+  'url.weight': 'w3', // urlWeight
+  'url.style': 's3', // urlStyle
+  'url.color': 'c5', // urlColor
+  'colors.background': 'c2', // borderColor
+  'colors.border': 'c1', // borderColor
+  'colors.link': 'c6', // lnkColor
+};
+
+let OrbitsoftAdapter = function OrbitsoftAdapter() {
+  let baseAdapter = new Adapter(ORBITSOFT_BIDDERCODE);
+
+  baseAdapter.callBids = function(params) {
+    let bids = params.bids || [];
+
+    for (let i = 0; i < bids.length; i++) {
+      let bidRequest = bids[i];
+      let callbackId = bidRequest.bidId;
+      let jptCall = buildJPTCall(bidRequest, callbackId);
```

Comment:
I see you are hitting the AppNexus endpoint. Is there a reason you cannot use an AppNexus alias instead of creating your own adapter?

Reply:
AppNexus has built-in enpoint URL for bid requesting. Our adapter sends bid request to various endpoint URL where customers install our product. Endpoint URl is defined by the requestUrl parameter and different for diffrent customers.
We also have additional options for requesting bids and ads.